io.github.baldeuniversel.am-okay.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It looks like you've uploaded this file to the application's repository, which is good.

However, the app ID you've chosen is `io.github.baldeuniversel.am_okay`, while the file is named `io.github.baldeuniversel.am-okay` (you're using the underscore character on one, and the dash character on the other).

Pick whichever ID you'd like, keep it consistent and use the same one everywhere.

After that, you can remove this file from here.